api/src/main/java/io/opentelemetry/trace/attributes/StringAttributeSetter.java

```diff
@@ -61,4 +61,14 @@ public String key() {
   public void set(Span span, @Nullable String value) {
     span.setAttribute(key(), value);
   }
+
+  /**
+   * Sets the attribute on the provided span builder.
+   *
+   * @param spanBuilder the span builder to add the attribute to
+   * @param value the value for this attribute
+   */
+  public void set(Span.Builder spanBuilder, @Nullable String value) {
+    spanBuilder.setAttribute(key(), value);
```

Comment:
would it make sense to return this, or the builder from this call? Or does that not help make a more fluent API?

Reply:
I guess it doesn't make any sense to return `this`, but would returning the Builder be at all useful?

Reply:
And, I see that @Oberon00 already made this comment above. sorry for the extra noise.

Reply:
`SemanticAttributes` are now used like this: `SemanticAttribute.HTTP_URL.set(span, value). No quick way to have fluent interface out of this.